asab/task.py

```
@@ -59,13 +59,19 @@ async def finalize(self, app):
 				self.PendingTasks.remove(task)
 			except asyncio.CancelledError:
 				self.PendingTasks.remove(task)
-			except Exception as e:
-				L.exception("Error '{}' during task service:".format(e))
+			except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)